examples/beginner/print_pretty.py

```diff
@@ -39,5 +43,8 @@ def main():
     pprint( (1/cos(x)).series(x, 0, 10) )
     print('\n')
 
+    pprint(a*(KroneckerProduct(b, c)))
```

Comment:
Modifying the examples is fine, but the test for this needs to go in the test file (sympy/printing/pretty/tests/test_pretty.py).

Reply:
I have added tests for this issue.
thanks for understanding noob